bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDBTest.java

```diff
@@ -78,4 +80,38 @@ public void testRocksDBInitiateWithConfigurationFile() throws Exception {
         assertEquals(1, familyOptions.maxWriteBufferNumber());
         rocksDB.close();
     }
+
+    @Test
+    public void testReadChecksumTypeFromBookieConfiguration() throws Exception {
+        ServerConfiguration configuration = new ServerConfiguration();
+        configuration.setProperty("dbStorage_rocksDB_checksum_type", "kxxHash");
```

Comment:
We can remove  `configuration.setProperty("dbStorage_rocksDB_checksum_type", "kxxHash");` to verify the default value is `kxxHash`.

Reply:
done